Sources/OpenTelemetrySdk/Metrics/Stable/LongHistogramMeterBuilderSdk.swift

```diff
@@ -1,41 +1,39 @@
 //
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
-// 
+//
 
 import Foundation
 import OpenTelemetryApi
 
-public class LongHistogramMeterBuilderSdk : LongHistogramBuilder, InstrumentBuilder {
-    
+public class LongHistogramMeterBuilderSdk: LongHistogramBuilder, InstrumentBuilder {
     var meterProviderSharedState: MeterProviderSharedState
     
     var meterSharedState: StableMeterSharedState
     
-    var type: InstrumentType = .histogram
+    let type: InstrumentType = .histogram
     
-    var valueType: InstrumentValueType = .long
+    let valueType: InstrumentValueType = .long
```

Comment:
These properties should be constant because it ties with the class itself. 
Besides, it should apply to other properties like `instrumentName`, `description` and `unit`